static/js/weather-components.js

```
@@ -619,23 +619,23 @@
         const temps = hourlyData.map(h => h.temp);
         const maxTemp = Math.max(...temps);
         const minTemp = Math.min(...temps);
         const tempRange = maxTemp - minTemp || 1;
-        
+
         let pathData = '';
         temps.forEach((temp, index) => {
             const x = (index / (temps.length - 1)) * width;
             const y = height - ((temp - minTemp) / tempRange) * height;
             pathData += (index === 0 ? 'M' : 'L') + x + ',' + y;
         });
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)